### ISSUE TITLE
Remarks section missing

### ISSUE THREAD

**Issue by jasonshave** [Azure/azure-sdk-for-net#20084]
Remarks section is empty

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8d9939c4-1d84-fcc3-cae6-928d900db405
* Version Independent ID: aaed33c7-5eb5-185f-7e09-c90853405fe4
* Content: [RuleFilter Class (Azure.Messaging.ServiceBus.Administration) - Azure for .NET Developers](https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.administration.rulefilter?view=azure-dotnet)
* Content Source: [xml/Azure.Messaging.ServiceBus.Administration/RuleFilter.xml](https://github.com/Azure/azure-docs-sdk-dotnet/blob/master/xml/Azure.Messaging.ServiceBus.Administration/RuleFilter.xml)
* Service: **azure**
* GitHub Login: @CamSoper
* Microsoft Alias: **casoper**

**Comment by jsquire** [comment]
This looks like an issue with the XML comments.  The `<item>` tag for a list is expected to have a child `<term>` and `<description>`, which is what the Docs [transform](https://github.com/docascode/ECMA2Yaml/blob/develop/ECMA2Yaml/ECMAHelper/Transform/ECMADocsTransform.xsl#L194) is keying on.    We'll need to do a scrub across the project to patch these up.

```csharp
    /// <summary>
    /// Describes a filter expression that is evaluated against a Message.
    /// </summary>
    /// <remarks>
    /// Filter is an abstract class with the following concrete implementations:
    /// <list type="bullet">
    /// <item><b>SqlRuleFilter</b> that represents a filter using SQL syntax. </item>
    /// <item><b>CorrelationRuleFilter</b> that provides an optimization for correlation equality expressions.</item>
    /// </list>
    /// </remarks>
    /// <seealso cref="SqlRuleFilter"/>
    /// <seealso cref="TrueRuleFilter"/>
    /// <seealso cref="CorrelationRuleFilter "/>
    /// <seealso cref="FalseRuleFilter"/>
    public abstract class RuleFilter : IEquatable<RuleFilter>
```

**Comment by jsquire** [comment]
Hi @jasonshave.  Thank you for your feedback and we regret that you're experiencing difficulties.   This is confirmed to be a bug in the XML doc comments in the code.  We'll get this fixed up.

**Comment by jsquire** [comment]
closed by #20117

**Comment by jsquire** [comment]
This has been fixed and the updated docs should be available with the pending April release of Service Bus.   I'm going to close this out; please feel free to reopen if you believe that further discussion is needed.
